airflow/providers/odbc/hooks/odbc.py

```diff
@@ -171,6 +171,8 @@ def clean_bool(val):  # pylint: disable=inconsistent-return-statements
                     return True
                 elif val.lower() == 'false':
                     return False
+                else:
+                    return val
```

Comment:
With this change we may be able to remove the pylint comment from L168, WDYT?

Reply:
There are different types (at least boolean and string) that could be returned by this function. Therefore (I suppose) the code analyser complains of this fact.
However, as I realized, it could be acceptable sometimes in dynamic languages, so we can conditionally return different data types.
In our case the clean_bool() function's result is used as a value in the dictionary and dictionaries itself can contain the keys/values of the different types. Therefore we may ignore this warning and remove the pylint comment from L168.
Please tell me if it does make sense for you.

Reply:
As the clean_bool is planned to be removed, the pylint comment will be cleaned as well.